app/assets/stylesheets/partials/_autocomplete.scss

```diff
@@ -36,4 +36,9 @@ $custom-combobox-input-border-width: 2px;
   @media screen and (min-width: $media-query-breakpoint) {
     right: calc(50% - #{$custom-combobox-input-border-width});
   }
+}
+
+.autocomplete__wrapper {
```

Comment:
this wont work, as it will mix the dropdown elments with the content on the page. Please check

Reply:
Removed this part